### DISCUSSION TITLE
MAUI projectreference

### DISCUSSION THREAD

**Discussion by qunshen** [oqtane/oqtane.framework#2659] [General] [answered]
I ntices that the Oqtane.Maui.csproj references two packages(Oqtane.Client and Oqtane.Shared). Generally, for development, it is better to refrence projects(Oqtane.Client and Oqtane.Shared) than packages. Are there other conderations for doing so?

**Comment by leigh-pointer** [comment]
@qunshen I really don't think there would be an issue referencing the projects.

**Comment by sbwalker** [comment, marked as answer]
In preparation for the 3.4.0 release the package references in the Oqtane.Maui project were upgraded to 3.4.0 - however the 3.4.0 packages have not yet been released to Nuget - which will result in broken references. So this might be causing you some difficulty if you are trying to use the code in the Dev branch at this moment in time - you would either need to reference the 3.3.1 packages or switch to project references. This is only a temporary issue until 3.4.0 is released - which will be soon as regression is near complete (https://www.oqtane.org is already dogfooding the latest bits).
